.github/actions/bot/index.js

```diff
@@ -0,0 +1,154 @@
+        return;
+    }
+    console.log(`Comment author is authorized: ${author}`);
+
+    const commands = parseCommands(uuid, payload, payload.comment.body);
+    if (commands.length === 0) {
+        console.log("No commands found in comment body");
+        return;
+    }
+    const uniqueCommands = [...new Set(commands.map(command => typeof command))];
+    if (uniqueCommands.length != commands.length) {
+        console.log("Duplicate commands found in comment body");
+        return;
+    }
+    console.log(commands.length + " command(s) found in comment body");
+
+    for (const command of commands) {
+        const reply = await command.run(author, github);
+        if (typeof reply === 'string') {
+            github.rest.issues.createComment({
+                owner: payload.repository.owner.login,
+                repo: payload.repository.name,
+                issue_number: payload.issue.number,
+                body: reply
+            });
+        } else if (reply) {
+            console.log(`Command returned: ${reply}`);
+        } else {
+            console.log("Command did not return a reply");
+        }
+    }
+}
+
+// parseCommands splits the comment body into lines and parses each line as a command.
+function parseCommands(uuid, payload, commentBody) {
+    const commands = [];
+    if (!commentBody) {
+        return commands;
+    }
+    const lines = commentBody.split(/\r?\n/);
+    for (const line of lines) {
+        const command = parseCommand(uuid, payload, line);
+        if (command) {
+            commands.push(command);
+        }
+    }
+    return commands
+}
+
+// parseCommand parses a line as a command.
+// The format of a command is `/NAME ARGS...`.
+// Leading and trailing spaces are ignored.
+function parseCommand(uuid, payload, line) {
+    const command = line.trim().match(/^\/([a-z\-]+)(?:\s+(.+))?$/);
+    if (command) {
+        return buildCommand(uuid, payload, command[1], command[2]);
+    }
+    return null;
+}
+
+// buildCommand builds a command from a name and arguments.
+function buildCommand(uuid, payload, name, args) {
+    switch (name) {
+        case "echo":
+            return new EchoCommand(uuid, payload, args);
+        case "ci":
+            return new CICommand(uuid, payload, args);
+        default:
+            console.log(`Unknown command: ${name}`);
+            return null;
+    }
+}
+
+class EchoCommand {
+    constructor(uuid, payload, args) {
+        this.phrase = args ? args : "echo";
+    }
+
+    run(author) {
+        return `@${author} *${this.phrase}*`;
+    }
+}
+
+class CICommand {
+    constructor(uuid, payload, args) {
+        this.repository_owner = payload.repository.owner.login;
+        this.repository_name = payload.repository.name;
+        this.pr_number = payload.issue.number;
+        this.comment_url = payload.comment.html_url;
+        this.uuid = uuid;
+        this.goal = "test";
+        if (args != null && args != "") {
```

Comment:
nit: a short line to document the behavior

```suggestion
        // "test" goal executes all CI stages by default
        if (args != null && args != "") {
```

Reply:
Done 👍 I intend to add a `/help` command in the short term as well